minecraft_servers/skypotion/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+    "server_name": "skypotion",
+    "nice_name": "SkyPotionEU",
+    "direct_ip": "skypotion.eu",

```

Comment:
Your server is not reachable.
We can only accept PRs for servers that are up and running.